HLTrigger/HLTanalyzers/plugins/HLTInfo.h

```diff
@@ -109,7 +111,7 @@ HLTInfo::HLTInfo(edm::ParameterSet const& pset, edm::ConsumesCollector&& iC, T&
 
 template <typename T>
 HLTInfo::HLTInfo(edm::ParameterSet const& pset, edm::ConsumesCollector& iC, T& module) : HLTInfo() {
-  hltPrescaleProvider_.reset(new HLTPrescaleProvider(pset, iC, module));
+  hltPrescaleProvider_ = std::make_unique<HLTPrescaleProvider>(pset, iC, module);
```

Comment:
What happened to the call to method reset()?

Reply:
`reset()` releases the old pointed-to object and takes ownership of a new one; assigning a new `unique_ptr` has the same effect

Reply:
thanks...